src/shared-components/BottomNavigation/index.tsx

```diff
@@ -0,0 +1,106 @@
+import HomeIcon from '@mui/icons-material/Home';
+import SyncAltOutlinedIcon from '@mui/icons-material/SyncAltOutlined';
+import DataSaverOffOutlinedIcon from '@mui/icons-material/DataSaverOffOutlined';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+import { styled } from '@mui/material/styles';
+import { Typography } from '@mui/material';
+import { useState } from 'react';
+import { colors } from 'src/styles/colors';
+
+const StyledDivBottomNavigation = styled(`div`)(({}: {}) => ({
+  display: 'flex',
+  width: '375px',
+  height: '70px',
+  alignItems: 'center',
+  justifyContent: 'space-around',
+}));
+
+const StyledIconButton = styled('div')<{ active: boolean }>(({ active }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '35px',
+  height: '48px',
+  alignItems: 'center',
+  color: active ? colors.blue[100] : colors.light[20],
```

Comment:
The color of the selected item is not blue in the design, primary color should be used (theme.palette.primary.main). You can change this component like this:
```

const StyledIconButton = styled(`div`)<{ active: boolean }>(
  ({ active, theme }) => ({
    ...   
    color: active ? theme.palette.primary.main : colors.light[20],
    ...
  }),
);
```